Python/Product/TestAdapter/TestContainerDiscovererProject.cs

```diff
@@ -274,6 +288,7 @@ class TestContainerDiscovererProject : ITestContainerDiscoverer, IDisposable {
             var pyProj = PythonProject.FromObject(e.Project);
             if (pyProj != null) {
                 pyProj.ProjectPropertyChanged -= OnTestPropertiesChanged;
+                pyProj.ActiveInterpreterChanged -= OnActiveInterpreterChanged;
```

Comment:
do we need to _packageManagerEventSink.Unwatch(project)  here?

Reply:
I don't think so. When any change requires refresh, we clear all. Then every project will add its interpreter factory package managers.

Reply:
ahh..  I need to look again if calling refresh is really necessary when a project unloads. for now this works